src/items/entities/history.entity.ts

```diff
@@ -0,0 +1,48 @@
+import { Account } from '../../account/entities/account.entity';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Item } from './item.entity';
+
+@Entity()
+export class History {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ nullable: true })
+  price: string;
+
+  @Column()
+  transactionType: string;
```

Comment:
You should use the type 'enum' for enums in models. You can put something like : `@Column({ type: 'enum', enum: TransactionType, default: TransactionType.Minted })`